internal/apiclient/pricing.go

```diff
@@ -35,12 +35,14 @@ func NewPricingAPIClient(ctx *config.RunContext) *PricingAPIClient {
 		currency = "USD"
 	}
 
-	rootCAs, _ := x509.SystemCertPool()
-	if rootCAs == nil {
-		rootCAs = x509.NewCertPool()
-	}
+	tlsConfig := tls.Config{} // nolint: gosec
```

Comment:
I'm assuming passing an empty `tls.Config` works the same as passing nil yeah?

Reply:
Yeah the pattern in the libraries is
```
	if t1.TLSClientConfig == nil {
		t1.TLSClientConfig = new(tls.Config)
	}
```